### DISCUSSION TITLE
Missing Data on QField & QGis but visible on "changes" on QField Cloud

### DISCUSSION THREAD

**Discussion by Notramus** [opengisch/QField#2918] [Q&A]
Hello! 

We have run into the issue that after uploading the data ("push changes") and subsequent synchronization via the QField-App leads to data loss in both: QField App and QGis on the PC.

If I check my project on the cloud I can find the loss data in "changes".  They show the status "applied" but they cannot be find in the project nor in my project files if I download it manually. 

What could I be missing?

Sincerely,
Notramus

**Comment by OBVYamaska** [comment]
I posted on the same matter 2 weeks ago, but still got no answer :(
I also wonder if there is a way to physically copy the datas from the device before pushing the changes online, to have a backup and to be able to investigate better the data loss afterward, but I can't find the QFieldCloud project on the device...

**Comment by suricactus** [comment]
Hey, @Notramus thank you for trying QFieldCloud BETA.

Can you please give me a hint about your project and the problematic changes? Can you share your project name and your change ids?

**Reply by Notramus** [reply]
Hey, @suricactus thank you for your help! I am sorry for the late reply, but hope that you can still help me to figure out what is going here. 
I have invited you to the project and given you admin rights. I am having trouble with following IDs.
<html>
<body>


c8825863 | lucasauguanno | 05/12/2022 4:04 p.m. | Applied |  
  | fabe0cdf | lucasauguanno | 05/12/2022 4:04 p.m. | Applied |  
  | 2d14822f | lucasauguanno | 05/12/2022 4:04 p.m. | Applied |  
  | 96d269b5 | lucasauguanno | 05/12/2022 4:04 p.m. | Applied |  
  | b90421ed | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 1a418c3d | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 818115f2 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 908cdd37 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 8916d153 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | a88bfd1d | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 0bcdfc52 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 0a3fba9e | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 5a714450 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 88cf69a0 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | b05df169 | lucasauguanno | 05/12/2022 4:03 p.m. | Applied |  
  | 3e06b1ff | lucasauguanno | 05/12/2022 4:03 p.m. | Applied
